packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -806,6 +807,7 @@ class DataGrid extends React.Component {
           className={cx('header-overflow-container')}
           ref={this.setHeaderOverflowContainerRef}
           onScroll={this.synchronizeHeaderScroll}
+          tabIndex="0"
```

Comment:
By adding tabIndex=0, we are allowing user to tab to this element. Is this desired behavior of this component?

Reply:
The DataGrid is so hands-on with the tabbing management for the cells that I don't think it will allow these containers to ever get focus. At least I can't get it to happen for this PR deployment. That might be an accessibility concern in and of itself, but it should at least alleviate this specific concern.